test/hdfs_test.py

```diff
@@ -32,6 +33,43 @@ class TestException(Exception):
 
 
 @attr('minicluster')
+class ConfigurationTest(MiniClusterTestCase):
+
+    def tezt_rename_dont_move(self, client):
```

Comment:
haha tezt? I have a minor preference for something less confusing like just `def rename_dont_move`